### ISSUE TITLE
What is the intention of using naturalWidth / naturalHeight on the wild 

### ISSUE THREAD

**Issue by loonybear** [GoogleChrome/lighthouse#6845]
Hi, I am working on shipping [intrinsicSize attribute](https://github.com/ojanvafai/intrinsicsize-attribute). intrinsicSize will override the value of naturalWidth / naturalHeight. I realized in [this issue](https://github.com/ojanvafai/intrinsicsize-attribute/issues/10) that lighthouse is using naturalWidth / naturalHeight, and I would like to understand your intention. Perhaps if needed we could support the intention of getting the image's physical dimension by shipping new set of attribute: physicalWidth/ physicalHeight. 

Thanks

@yoavweiss @tabatkins

**Comment by patrickhulce** [comment]
tl;dr I think `phyiscalWidth/physicalHeight` would be useful to us and is what we actually want.

We use `natural*` properties for two purposes:

1. Checking if images are too large for their display size (i.e. are they serving responsive, mobile-sized images to mobile devices)
2. Checking if images are displayed at the same aspect ratio as their source (i.e. not accidentally stretching a portrait image into a landscape box)


1 is the far more important use case for us and based on https://github.com/ojanvafai/intrinsicsize-attribute/issues/10 it seems like we're kind of cheating at the moment and `physicalWidth` / `physicalHeight` is what we're actually after.

Bullet 1 in @yoavweiss's comment pretty much captures our situation https://github.com/ojanvafai/intrinsicsize-attribute/issues/10#issuecomment-448016847

**Comment by loonybear** [comment]
Great. So seems like physical(Width|Height) is the way to go then. 

I am working on some [feature policies for image performance](https://github.com/loonybear/feature-policy/blob/patch-1/policies/optimized-images.md). I am curious to learn about your strategies for #1 (as how you set up threshold and etc). Also #2 could be benefited from using intrinsicSize and maybe [unsized-media](https://github.com/loonybear/feature-policy/blob/patch-1/policies/unsized-media.md) policy.

**Comment by patrickhulce** [comment]
1 is currently based on % of pixels used, so we compute the total area needed to display the image at the current `clientWidth`/`clientHeight` and device's DPR. Anything more than that is "waste". We allow 2KB of waste since the breakpoints are never going to be perfect. 

https://github.com/GoogleChrome/lighthouse/blob/0f46ebfdbf50d1a5b1f75659db126b2a8de35d4e/lighthouse-core/audits/byte-efficiency/uses-responsive-images.js#L60-L65

I would love to see a lot of LH audits "graduate" to feature policy in the browser that we can easily piggyback on :D

**Comment by paulirish** [comment]
We're eager to migrate our audits to feature policies. Let us know when we can!
